server.js

```diff
@@ -53,6 +54,7 @@ app.use('/api', router);
 // Get routes from files in app/routes
 app.use('/users', users);
 app.use('/households', households);
+app.use('contacts', contacts);
```

Comment:
Need to add a / before the first contacts:
app.use('/contacts', contacts);

Reply:
fixed it, thanks!